### ISSUE TITLE
[1.3.0.1] Other plugins sometimes stop working after switching servers

### ISSUE THREAD

**Issue by spradlin-dev** [Casimir255/SeamlessClientPlugin#20]
I've seen other plugins stop working *sometimes* after switching servers within a cluster.  Most of the time the plugin that breaks is Tool Switcher, but I've also seen the same issue with Block Picker.  Those are the only two other in-game plugins I use.  When I uninstalled Seamless for a few weeks, those plugins never stopped working, and after re-installing, they're acting up again.

**Comment by Casimir255** [comment]
Yeah, seamless can do that. Nothing I can do to fix either. Just how seamless currently works and some things don't play nice. Mods, and local plugins
